conf/debug.yaml

```diff
@@ -14,6 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+enable_debug: false
```

Comment:
```suggestion
basic:
  enable: false
```
would be better?

Reply:
Looks better, but this `enable` will conflict with https://github.com/apache/apisix/blob/b52f95b53c689cfd2fd0f4cbf0fefedd4cce9b5e/t/debug/hook.t#L34

I don't know how to modify this with perl.

Reply:
Let me do it for you 😃